prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -37,9 +53,15 @@ Component "{{ .Env.XMPP_INTERNAL_MUC_DOMAIN }}" "muc"
     }
     storage = "none"
     muc_room_cache_size = 1000
+    {{ if .Env.ENABLE_TOKEN_AUTH }}
+    modules_enabled = { "token_verification" }
+    {{ end }}
 
 Component "{{ .Env.XMPP_MUC_DOMAIN }}" "muc"
     storage = "none"
+    {{ if .Env.ENABLE_TOKEN_AUTH }}
+    modules_enabled = { "token_verification" }
+    {{ end }}
```

Comment:
One more question: Is it right to add this module on both `XMPP_INTERNAL_MUC_DOMAIN ` and `XMPP_MUC_DOMAIN `?

Reply:
It should only be needed in XMPP_MUC_DOMAIN